fontforge/scripting.c

```diff
@@ -2381,10 +2414,11 @@ static void bImport(Context *c) {
 	ok = FVImportMult(c->curfv,filename, back, bf_ttf);
     else if ( format==fv_pk )
 	ok = FVImportBDF(c->curfv,filename,true, back);
-    else if ( format==fv_image || format==fv_eps || format==fv_svg || format==fv_pdf )
-	ok = FVImportImages(c->curfv,filename,format,back,flags);
+    else if (    format==fv_image || format==fv_eps || format==fv_svg
```

Comment:
nit - not a fan of this style of putting the `||` on the next line and offsetting the first by that

Reply:
I find it makes the expressions much more readable.

Reply:
well it's definitely a unique coding style, usually you just have the || at the end of the line and align on the start of the bracket instead

Reply:
This style of formatting communicates a higher level of commitment than that one because any logical connector at any level of nesting can wind up at the end of the line and be formatted that way. For example:

```
    if ( s==spline_to && (( t_fm<=t_to && !backward ) ||
         (t_fm>=t_to && backward))) {
```
vs
```
    if (    s==spline_to
         && (( t_fm<=t_to && !backward ) || (t_fm>=t_to && backward))) {
```
With the first you have to scan across counting parentheses to make sense of the logical structure. With the second I as the author have done the scanning and am communicating the structure to you the reader. 

Similarly with this from `utanvec.c`:
```
    if ( ys1d.a!=0 && (   Within16RoundingErrors(ys1d.a+ys1d.d,ys1d.d)
                       || Within16RoundingErrors(ys1d.a+yto,yto)))

```

Reply:
Typically in that case you'd just indent further on the second line to indicate a continuation of the subcondition.

Reply:
#830 is open. The project could arrive at a consensus style for new code; it hasn't done so. Until then I recommend against anyone presenting their own preferences as objectively better in code review contexts. I cannot believe you're unaware that there is wide disagreement about these questions.